### PR TITLE
chore(travis): make browserstack unit tests allowed failures

### PR DESCRIPTION
**Pull request by juliemr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ env:
     - LOGS_DIR=/tmp/angular-build/logs
     - BROWSER_PROVIDER_READY_FILE=/tmp/browsersprovider-tunnel-ready
 
+matrix:
+  allow_failures:
+    - env: "JOB=unit BROWSER_PROVIDER=browserstack"
```
Comment (caitp) [inline .travis.yml]: we could do this for the E2E test failures too, though they haven't been having the same issues

**Comment by caitp** [comment]
that comments not a blocker, so lgtm --- but it might save some headaches in the future to apply it to e2e tests too

**Comment by juliemr** [comment]
Discussed with @vojtajina and we want to just do unit tests for now, since e2e tests have been stabler.

**Comment by caitp** [comment]
sgtm

**Comment by juliemr** [comment]
Merged to master as 40cb57c8f6205eb4062b150f829b066d72f9f59c
Merged to 1.3.x as a06193f97b8ff24c76433c74eb2f6296ef937cb4

**Pull request merged**